Provis.Core/Services/WorkspaceService.cs

```diff
@@ -265,5 +266,29 @@ public async Task<WorkspaceInfoDTO> GetWorkspaceInfoAsync(int workspId, string u
 
             return workspace;
         }
+
+        public async Task<List<UserInviteInfoDTO>> 
+            GetWorkspaceActiveInvitesAsync(int workspId, string userId)
+        {
+            var workspace = _workspaceRepository
+                .Query()
+                .FirstOrDefault(x => x.Id == workspId);
+
+            if (workspace == null)
+            {
+                throw new HttpException(System.Net.HttpStatusCode.NotFound, 
+                    "Workspace with this ID not exist");
+            }
+
+            var invitesList = await _inviteUserRepository
+                .Query()
+                .Include(x => x.FromUser)
+                .Where(x => x.WorkspaceId == workspId && x.FromUserId == userId && x.IsConfirm == null)
+                .ToListAsync();
```

Comment:
Logical miss we should get all invite from workspace when user role is owner for current